addon/components/input-credit-card-expiration.js

```diff
@@ -48,33 +48,30 @@ export default TextField.extend({
 
   keyPress: function(e) {
     var digit = String.fromCharCode(e.which);
-    
+
     if (!isDigitKeypress(e)) {
       return false;
     }
-    
+
     var el = this.$();
 
     if (hasTextSelected(el)) {
       return true;
     }
-    
+
     var value = el.val() + digit;
     return inputValid(value);
   },
 
   value: computed('month', 'year', {
     get() {
-      var month = this.get('month');
-      var year  = this.get('year');
-
-      return formatters.formatExpiration(month, year);
+      return formatters.formatExpiration(this.get('month'), this.get('year'));
     },
-
     set(key, value) {
-      var parsed = parseInput(value);
-      var month = parsed[0];
-      var year = parsed[1];
+      const parsed = parseInput(value);
```

Comment:
Yes they change.  I am not sure how babble polyfills the spec but `const` should be immutable, and thus fail to update the the month and year.